download/download.go

```diff
@@ -84,6 +84,9 @@ func download(path, dest string, firstAttempt bool) (root *vcs.RepoRoot, err err
 		// may have been rebased; we delete the directory, then try one more time:
 		log.Printf("Failed to update %q (%v), trying again...", root.Repo, err.Error())
 		err = os.RemoveAll(fullLocalPath)
+		if err != nil {
+			log.Printf("Failed to delete directory : %s", fullLocalPath)
```

Comment:
nitpick but could you remove the space before the `:` here? 

log.Printf("Failed to delete directory: %s", fullLocalPath)

Reply:
Done